### ISSUE TITLE
Cannot download contour data from earthexplorer.usgs.gov using phyghtmap

### ISSUE THREAD

**Issue by sekocabas** [der-stefan/OpenTopoMap#253]
Hello,

I am trying to create a garmin map file for US north-east region. It seems like the http interface that `phyghtmap` (v 2.21) uses to access `earthexplorer.usgs.gov` is no longer valid and as a result I am getting file not found errors from `phyghtmap` run. I created earthexplorer account and verified that I can login. 

Are there any alternatives to downloading contour data to create custom garmin maps for US regions using OpenTopoMap settings? If it is the case that `phyghtmap` is currently working, can you share with me the command parrameters necessary to download the contours for the poly file available at https://download.geofabrik.de/north-america/us-northeast.html ?

**Comment by alexbodn** [comment]
i think you should register to earth explorer and 
confirm their registration request sent to you by mail.

alex

**Comment by blacklion** [comment]
> I am trying to create a garmin map file for US north-east region. It seems like the http interface that `phyghtmap` (v 2.21) uses 
You should use 2.22+ (released several days ago) and have valid login and password for the USGS service.

**Comment by sekocabas** [comment]
Thank you for the tips. I downloaded `phyghtmap` (v2.23) and from the changelog I see the following.
    
    Updated to work with new earthexplorer interface: SRTM[1|3] v3 data can be downloaded again.

On the other hand, North American Garmin files are now available on the https://garmin.opentopomap.org/ website (thank you!). I therefore am abandoning my attempt to generate the `img` files from scratch. I presume these `img` files use the 3 arc-second SRTM3 database, though I may be wrong. If so, and if I ever have a need to generate SRTM1 versions with 1 arc-second resolution, I will revisit contour `img` file compilation using the information provided at https://wiki.openstreetmap.org/wiki/OSM_Map_on_Garmin/Contours_using_phygtmap and the contour options and styles defined at https://github.com/der-stefan/OpenTopoMap/tree/master/garmin

Closing this issue.
